.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - run:
         |
-          sudo wget https://github.com/ethereum/solidity/releases/download/v0.6.12/solc-static-linux -O /usr/local/bin/solc
```

Comment:
Are we even using circle CI? If not, we should setup Github actions. Should be easy now that we have hardhat doing everything.

Reply:
Nope I don't believe so, I think this is Compound's. I'll setup GH actions.

Reply:
Done! I set `continue-on-error: true` for the coverage job for now, because of the issue with forking.